clash-vexriscv-sim/data/vexriscv_chain_sim.cfg

```diff
@@ -9,32 +9,27 @@
 
 # See vexriscv_sim.cfg for more information on each step taken.
 
-adapter driver jtag_tcp
-adapter speed 64000
+adapter driver remote_bitbang
```

Comment:
I get that we switched drivers, but could you elaborate on _why_ we switched drivers? In the commit body  of "[Switch to openocd-vexriscv and use remote-bitbang](https://github.com/clash-lang/clash-vexriscv/pull/45/commits/556b479091a8d8ac9febb709417f40308b67c86d)" please.

Reply:
`jtag_tcp` is not a normal openocd adapter, it was custom added to `openocd-vexriscv` and is very similar to the existing `remote-bitbang` target. I'll add it to the commit message